app/lib/package.rb

```diff
@@ -14,11 +14,20 @@ class Package
   XPATH_TIPS = %w[//training:tip/@id
     //tract:header/@training:tip
     //tract:call-to-action/@training:tip].freeze
+  TRANSLATION_FILES_PATH = "/translations/files/"
```

Comment:
```suggestion
  TRANSLATION_FILES_PATH = "translations/files/"
```

The leading `/` was generating a double-slash in S3. here is a sample file on stage:
```
https://cru-mobilecontentapi-staging.s3.amazonaws.com//translations/files/9c9715c5abb15fba40fde219fe8c89e7c5442235c397ee1e7bcc5a0fadc9c821.jpg
```